src/main/java/org/arkecosystem/crypto/transactions/types/Transaction.java

```diff
@@ -141,4 +129,13 @@ public HashMap toHashMap() {
     public boolean hasVendorField() {
         return false;
     }
+
+    private Signer signer() {
+        return this.version > 1 ? new SchnorrSigner() : new ECDSASigner();
```

Comment:
@faustbrian @ItsANameToo Do you think this is needed? Or do we only sign v2s now?

Reply:
no need, we only sign v2 with the SDK. the ecdsa signing is only relevant for message signing if I'm not mistaken (tl;dr remove the check and default to schnorr)